BuildDetails.md

```diff
@@ -0,0 +1,55 @@
+
+### typescript
+#### typescript:base
+This task compules each .ts file in the Source directory into a .js file in the same directory.
+#### typescript:distribution
+The typescript:distribution task translates the project's main .ts file into a .js file that is placed in the Distribution directory and concatenated with the version number of the project.
+
+See [this](https://github.com/k-maru/grunt-typescript) page for more information on typescript.
+
+### copy
+This task copies files and folders from the Source directory into the Distribution directory. 
+* It copies the project's main .ts file into Distribution, concatenating the project version to the name, and does the same to the accompanying .d.ts file. 
+* It also copies the file created by typescript:distribution to the directory in Distribution named after the project name and version. 
+* The References directory in Source is copied to the Distribution directory, too. 
+* The README.md and LICENSE.txt files are both copied into Source and Distribution. 
+* The last things copied into the Distribution directory are the Source/Fonts, Source/Sounds, and Source/Theme directories.
+
+See [this](https://github.com/gruntjs/grunt-contrib-copy) page for more information on the task.
+
+### uglify
+The uglify:dist task minifies the .js files. The files are compressed into two files: One of them contains the file created when typescript:distribution was run along with all .js file in the settings and maps directories while the other file minifies index.js. A sourcemap is created for each of these new files. See [this](https://github.com/gruntjs/grunt-contrib-uglify) page for more information on uglify.
+
+### cssmin
+cssmin:dist minifies .css files. It compresses index.css and creates a sourcemap for the newly compressed file. Both of these files can be found in the Distribution directory. See [this](https://github.com/gruntjs/grunt-contrib-cssmin) page for more information on the task.
+
+### preprocess
+In order to preprocess the file created in the typescript:distribution task, preprocess:dist is run. This task will carry out the directives stated at the top of the file. See [this](https://github.com/jsoverson/grunt-preprocess) page for more information on preprocess.
+
+### processhtml
+The processhtml:dist task processes commands given in in Source/index.html. The commands specify what files need to be built. The task builds index.min.css, the project's main .min.js file, and the index.min.js. It also removes the block of code that imports all of the .js files, making the minified index.html file much smaller. See [this](https://github.com/dciccale/grunt-processhtml) page for more information on the task.
```

Comment:
On first read-through I couldn't figure out what you meant by commands. That should be explained.